### PR TITLE
Binary_Search.cpp

### PR DESCRIPTION
**Pull request by Harshdev625**
Added Binary Search in C++ Language.

### CODE REVIEW

**Comment by deniss-eh** [comment]
please make it more cool. by adding some information and comments in programs. like when you are taking input from user add some line so user can know what are program is asking for . if do these changes. i will be happy to merge your pr :)

**Review by deniss-eh — CHANGES_REQUESTED** [review]
https://github.com/deniss-eh/DSA-HACKTOBERFEST/pull/5#issuecomment-1277970455

**Comment by Harshdev625** [comment]
Okay, I will do it immediately.

**Comment by Harshdev625** [comment]
Done.

File: C++/Binary_Search.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        else
+        {
+            start = mid + 1;
+        }
+    }
+    return -1;
+}
+int main()
+{
+    // Input the size of the array
```
Comment (deniss-eh) [inline C++/Binary_Search.cpp:31]: you can add here like cout << "Enter size of array";
so user who running the program can know what type of input you are asking for.

File: C++/Binary_Search.cpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    int n;
+    cin >> n;
+    // Create an Array of N integers
+    int *a = new int[n];
+    for (int i = 0; i < n; i++)
+    {
+        cin >> a[i];
+    }
+    // Input the value of x you want to search
```
Comment (deniss-eh) [inline C++/Binary_Search.cpp:40]: here too

**Comment by deniss-eh** [comment]
Your program looks good . For Now Im merging the PR but remember this in future. Happy hacking :)

**Pull request merged**